lndgrpc/client.py

```diff
@@ -16,11 +16,107 @@ def init_wallet(self):
     def generate_seed(self):
         raise NotImplementedError
 
+    #WALLETUNLOCKERRPC
+    @handle_rpc_errors
+    def unlock_wallet(self, **kwargs):
+        # .lnrpc
+        request = walletunlocker.UnlockWalletRequest(
+            **kwargs,
+            # wallet_password=<bytes>,
+            # recovery_window=<int32>,
+            # channel_backups=<ChanBackupSnapshot>,
+            # stateless_init=<bool>,
+        )
+        try:
+            response = self._walletunlocker_stub.UnlockWallet(request)
+            return response
+        except Exception as e:
```

Comment:
Can we try put a more target exception here if there is a known expeted error - maybe we can wrap it in a nicer error with a message as you have below. But I would avoid catching all errors. Or just let it fail and let the caller decide how to handle the errors. Thoughts?